xbmc/settings/windows/GUIWindowSkinSettings.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ *      Copyright (C) 2005-2017 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "GUIWindowSkinSettings.h"
+
+#include "addons/Skin.h"
+#include "guilib/WindowIDs.h"
+
+
+CGUIWindowSkinSettings::CGUIWindowSkinSettings(void)
+: CGUIWindow(WINDOW_SKIN_SETTINGS, "SkinSettings.xml")
+{
+  m_loadType = KEEP_IN_MEMORY;
```

Comment:
What is the default load type in `CGUIWindow`? If it's not `KEEP_IN_MEMORY` this might change behaviour.